### ISSUE TITLE
Un-mute not working correctly

### ISSUE THREAD

**Issue by jessedobbelaere** [ArtemGordinsky/Spotifree#72]
When Spotifree detects that the ad has finished, the volume goes back up but only to about 15%, which is almost unhearable. 
This is only started happening recently (maybe due to a Spotify update?). I already tried updating Spotifree to the latest version

<img width="198" alt="screen shot 2016-01-06 at 14 28 10" src="https://cloud.githubusercontent.com/assets/1352979/12143568/f0b5ac6c-b481-11e5-894e-f5cf7f851537.png">

**Comment by EneasR** [comment]
The next update is completely rewritten. I think it will fix this.
Just wait until it gets published.

**Comment by jessedobbelaere** [comment]
Hmm this is weird. I completely removed spotifree and spotify using Appcleaner. Rebooted. Installed spotify. Played a couple of songs and suddenly the ads get muted, out of nowhere. No spotifree app or service running, it's uninstalled

**Comment by EneasR** [comment]
Have you already tried terminating every Spotifree process using the Activity Monitor?

**Comment by jessedobbelaere** [comment]
Indeed, removed every trace and rebooted and checked activity monitor. Will check it out some more tonight. Strange wizardry. Any things I can check to see if spotifree is still involved, besides checking the process? 

`ps aux | grep spotifree` shows that there are no processes running, and still spotify gets muted.

**Comment by jessedobbelaere** [comment]
So my ads still get muted 😐Summary of the things I did:

* Removed Spotify and Spotifree (including every trace in /Library, using Appcleaner app)
* Rebooted a dozen times. Did a fresh install of Spotify. 
* Checked activitymonitor for suspicious processes, none are spotify or spotifree related. Don't see anything with `ps aux | grep spotifree` too.
* Did a `sudo find / -name '*spotifree*'` and `sudo find / -name '*eneas*'` to find files left on my hdd, none found.
* Never tried any other mute app (do other mute apps exist?). 

Is there some way that I can check if the 0.3s polling is still done, to find the cause which "thing" is dispatching the mute spotify action? Could it be a Spotify bug?

**Comment by ArtemGordinsky** [comment]
Haven't heard of any such bugs or features in Spotify, unfortunately.
Can we look at your process list? Something must be hiding there. ```ps aux``` output would do.

**Comment by jessedobbelaere** [comment]
Ok I figured out the issue. I have a mac mini at my parent's place with spotify installed and logged into my account. Spotifree was running there too. 

So when I play spotify music at work on my macbook pro without spotifree installed, the ads would get muted by the Spotifree app running at my parents place on the mac mini. 

Thanks man, I'll wait until the next update of Spotifree to fix this volume issue that is set to 1/5 after un-mute. :-)
